src/app/components/cds/buttons-checkboxes-chips/ButtonMain.js

```diff
@@ -1,40 +1,77 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import cx from 'classnames/bind';
 import Button from '@material-ui/core/Button';
 import styles from './ButtonMain.module.css';
 
 // FIXME: Refactor using native button instead of MUI Button
 const ButtonMain = ({
-  label,
-  variant,
-  disabled,
-  customStyle,
   buttonProps,
+  customStyle,
+  disabled,
+  iconCenter,
+  iconLeft,
+  iconRight,
+  label,
   onClick,
+  size,
+  theme,
+  variant,
 }) => (
   <Button
-    className={styles.buttonMain}
+    className={cx(
+      [styles.buttonMain],
+      styles[`theme-${theme}`],
+      {
+        [styles.sizeDefault]: size === 'default',
```

Comment:
This is a cool pattern! It took me a minute to puzzle it through but it's clear what it does and very clean